web/src/screens/filters/details.tsx

```diff
@@ -319,7 +318,51 @@ export default function FilterDetails() {
   );
 }
 
-export function General() {
+export function General({ values }: AdvancedProps){
+
+  const handleExportJson = () => {
+    const filteredValues = { ...values };
+    delete filteredValues.id;
+    delete filteredValues.name;
+    delete filteredValues.indexers;
+    delete filteredValues.actions;
+    delete filteredValues.external_script_enabled;
+    delete filteredValues.external_script_cmd;
+    delete filteredValues.external_script_args;
+    delete filteredValues.external_script_expect_status;
+    delete filteredValues.external_webhook_enabled;
+    delete filteredValues.external_webhook_host;
+    delete filteredValues.external_webhook_data;
+    delete filteredValues.external_webhook_expect_status;
+  
+    const json = JSON.stringify(filteredValues, null, 2);
```

Comment:
Something like this could maybe be nice?
```suggestion
    const json = JSON.stringify(filteredValues, null, 2);
        const json = JSON.stringify({
      title: "Exported autobrr filter",
      filter: filteredValues
    }, null, 2);
```